frontend/src/js/controllers/featuredChallengeCtrl.js

```diff
@@ -129,6 +129,13 @@
                 onSuccess: function(response) {
                     var details = response.data;
                     vm.leaderboard = details.results;
+                    for (var i=0; i<vm.leaderboard.length; i++) {
+                        var dateTimeNow = moment(new Date());
+                        var submissionTime = moment(vm.leaderboard[i].submission__submitted_at);
+                        var duration = moment.duration(dateTimeNow.diff(submissionTime));
+                        var hours = duration.asHours();
```

Comment:
I think it won't be ideal to show the number of hours since if the submission has been done for say 2 weeks before then it will be shown as `336 hours ago` which is not good. You might want to modify the function to return `X minutes ago` or `X hour(s) ago` or `X day(s) ago` or `X week(s) ago` or `X month(s) ago` depending on the duration.